girder/api/v1/file.py

```diff
@@ -254,8 +262,8 @@ def readChunk(self, upload, offset, params):
         .param('contentDisposition', 'Specify the Content-Disposition response '
                'header disposition-type value.', required=False,
                enum=['inline', 'attachment'], default='attachment')
-        .param('extraParameters', 'Arbitrary data to send along with the download request.',
-               required=False)
+        .jsonParam('extraParameters', 'Arbitrary data to send along with the download request.',
```

Comment:
I'm not sure what the ramifications are of changing from `param` to `jsonParam` will be.  This switching the extra parameters from a string to a parsed json object.  The only place I see it used before this is in downloads.